packages/@pollyjs/core/src/server/handler.js

```diff
@@ -11,7 +13,9 @@ export default class Handler extends Map {
     super();
 
     this.set('config', {});
+    this.set('defaultOptions', {});
```

Comment:
Curious why the name defaultOptions?

Reply:
Its more future proofing than anything else. Will allow us to add more API to modify default options passed to the interceptor/events.